### DISCUSSION TITLE
404 on auth

### DISCUSSION THREAD

**Discussion by kevincador** [trakt/trakt-api#382] [Q&A] [answered]
Hi 👋,
Any idea why the login may fail with a 404 when a user tries to auth?
Same thing works for the same user on the iOS version of the app but not on the macOS version. Maybe Safari doing something Safari mobile is not??
Thanks for the help,
Kevin

![image](https://user-images.githubusercontent.com/146673/202031347-de97e286-8029-403b-b29e-b4f10a8f9105.jpeg)

**Comment by rectifyer** [comment]
Hmm, what is the specific URL it's loading? The page is the same for desktop and mobile, it's just responsive CSS.

**Comment by kevincador** [comment, marked as answer]
It's solved for the user. 

It was a Safari Extension that was running (Noir). The way [ASWebAuthenticationSession](https://developer.apple.com/documentation/authenticationservices/aswebauthenticationsession) works on iOS (via a special WebView) and on Mac (via a browser window) is probably also part of the issue. I think the extensions are not active on iOS while they are on Mac. 

I don't know if it is possible to ask Safari to run without extensions on certain pages (locally or on the page itself).

I'll add some guidance in app for these kind of issues.

**Comment by kevincador** [comment]
FYI @rectifyer here's the URL that gave the 404 for the user: https://trakt.tv/auth/dist/noirhelper.js
I think the extension https://getnoir.app is doing something sketchy and not sure you can do something to fix that.
The user tells me he got this after logging in, so, no problem until he hits the log in button on the auth page.

**Reply by rectifyer** [reply]
We could handle that path and just ignore it, but ultimately seems like a bug with that extension that needs fixing.
